main.go

```diff
@@ -193,6 +198,15 @@ func findFiles(patterns []string) (_ []string, err error) {
 	return sortedFiles, err
 }
 
+func modify(filename string, content []byte) error {
+	return ioutil.WriteFile(filename, content, 0644)
+}
+
+func preview(filename string, originalContent []byte, modifiedContent []byte, comment string, stderr io.Writer) error {
+	fmt.Fprintf(stderr, comment+"\n")
+	return diff.Text(filename, "changed File", originalContent, modifiedContent, stderr)
```

Comment:
I think `changed File` is not informative here. Can we change it to the `filename` instead?